docs/book/src/quinn/data-transfer.md

```diff
@@ -0,0 +1,172 @@
+# Data Transfer
+
+In the [previous chapter](set-up-connection.md) we characterized how to set up an [Endpoint][Endpoint]
+and then get access to a [NewConnection][NewConnection] instance.
+Now we will continue with the subject of sending data over this connection.
+
+## Multiplexing
+
+A QUIC stream can be compared to a TCP stream, eventhough if it is based on UDP. 
+The big difference is that you are not limited to a single stream. 
+You can open multiple streams between two peers, this is also called multiplexing.
+
+Stream multiplexing can have a significant positive effect on application performance if the resources allocated to streams are properly prioritized.
+Currently multiplexing it is used in a protocol like HTTP/2, but unlike HTTP multiplexing, QUIC does not automatically multiplex.
```

Comment:
This comparison doesn't make much sense to me. HTTP isn't really a stream-oriented protocol, it's a request-response protocol.

Reply:
I am trying to shed light on the comparison between how multiplexing works in HTTP protocol and in quic. In quic it is not done automatically were as in HTTP it, right?

Reply:
I am going to rewrite this part a bit differently. I think there is some unnecessary info here.